src/main/java/com/bullhorn/dataloader/util/TemplateUtil.java

```diff
@@ -54,14 +55,33 @@ private void writeToCsv(String entity, ArrayList<String> headers, ArrayList<Stri
     }
 
     protected void populateDataTypes(String entity, Set<Field> metaFieldSet, ArrayList<String> headers, ArrayList<String> dataTypes) {
```

Comment:
Why is this refactoring needed? If you are passing the String entity, it would be time wasted refactoring to essentially do the same thing, but with a different data type, with a flow that only includes TemplateUtil and TemplateService. loadClass gets any relevant information from REST-SDK.